packages/ui-components/src/components/HeaderContainer/HeaderContainer.component.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Juno contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from "react"
+
+const headerContainerStyles = `
+  jn-flex
+  jn-flex-col
+  jn-sticky
+  jn-top-0
+  jn-z-50
+  jn-shadow-lg
+  jn-bg-theme-global-bg
+`
+
+export const HeaderContainer = ({
+  fullWidth = false,
+  className = "",
+  children = null,
+  ...props
+}: HeaderContainerProps) => {
```

Comment:
**(Blocking Improvement)**

To improve consistency, readability and type safety 🙏: 

```suggestion
export const HeaderContainer: React.FC<HeaderContainerProps> = ({
  fullWidth = false,
  className = "",
  children = null,
  ...props
}) => {
```